pyxem/utils/io_utils.py

```diff
@@ -181,23 +229,40 @@ def load_mib(mib_filename, reshape=True):
     data_pxm.metadata.Signal.exposure_time = data_dict['exposure time']
     data_pxm.metadata.Signal.frames_number_skipped = data_dict['number of frames_to_skip']
     data_pxm.metadata.Signal.flyback_times = data_dict['flyback_times']
-
-    if data_pxm.metadata.Signal.signal_type is 'TEM' and data_pxm.metadata.Signal.exposure_time is not None:
-        print('This mib file appears to be TEM data. The stack is returned with no reshaping.')
-        return data_pxm
-    try:
-        if reshape:
-            # If the exposure time info not appearing in the header bits use reshape_4DSTEM_SumFrames
-            # to reshape otherwise use reshape_4DSTEM_FlyBack function
-            if data_pxm.metadata.Signal.exposure_time is None:
-                data_pxm = reshape_4DSTEM_SumFrames(data_pxm)
-            else:
-                data_pxm = reshape_4DSTEM_FlyBack(data_pxm)
-    except TypeError:
-        raise ValueError('Reshaping did not work. Get the stack by passing reshape=False')
-    except ValueError:
-        raise ValueError('Reshaping did not work. Get the stack by passing reshape=False')
-
+    if reshape:
+    # only attempt reshaping if it is not already reshaped!
```

Comment:
Can a mib file already be reshaped?

Reply:
In case of using the h5 file loading route, I reshape after loading in the `h5stack_to_pxm` method. So here I am just checking the shape to not redo the reshaping.